server/controllers/EmailVerifyController.js

```diff
@@ -0,0 +1,96 @@
+import crypto from 'crypto';
+import jwt from 'jsonwebtoken';
+import db from '../database/models';
+import sendVerifyEmailMessage from './helpers/emailSender';
+
+const { User } = db;
+const secret = process.env.SECRET_KEY;
+const secret2 = process.env.SECRET_KEY2;
+
+/**
+ * Class representing email verification
+ */
+class EmailVerifyController {
+  /**
+   * Confirm a user email
+   * @param {*} req - Request object
+   * @param {*} res - Response object
+   * @param {*} next - Next function
+   * @returns {object} user - User object
+   */
+  static confirmationPost(req, res, next) {
```

Comment:
function name `confirmationPost`???

Reply:
It has been changed to emailVerification